__tests__/src/components/common/NationActionButton.js

```diff
@@ -1,46 +1,44 @@
-import NationActionButton from '../../../../src/components/common/NationActionButton';
 import React from 'react';
 import renderer from 'react-test-renderer';
-import AssetsImage from '../../../../src/global/AssetsImages';
 import { shallow } from 'enzyme/build/index';
 import { TouchableOpacity } from 'react-native';
+import NationActionButton from '../../../../src/components/common/NationActionButton';
+import AssetsImage from '../../../../src/global/AssetsImages';
 
 describe('NationActionButton component tests', () => {
   test('NationActionButton renders correctly', () => {
-    const tree = renderer.create(<NationActionButton
-      iconSource={AssetsImage.Actions.chat}
-      title='Chat'
-      disable
-    />).toJSON();
+    const tree =
+      renderer.create(<NationActionButton
+        iconSource={AssetsImage.Actions.chat}
+        title='Chat'
+        disable
+      />).toJSON();
     expect(tree).toMatchSnapshot();
   });
 
   test('NationActionButton press works', () => {
     const onPress = jest.fn();
-    const wrapper = shallow(<NationActionButton onPress={onPress} enabled />);
+    const wrapper =
+      shallow(<NationActionButton
+        iconSource={AssetsImage.Actions.chat}
+        onPress={onPress}
+        enabled
+      />);
     expect(wrapper).toMatchSnapshot();
     const render = wrapper.dive();
     render.find(TouchableOpacity).simulate('press');
     expect(onPress).toHaveBeenCalledTimes(1);
   });
 
-  test('NationActionButton press disabled', () => {
```

Comment:
Is it deleted intentionally? Looks like a test that should be there.

Reply:
Just added the missing test and updated the snapshots